examples/ac_asg/main.tf

```diff
@@ -256,7 +287,7 @@ module "ac_asg" {
   depends_on = [
     module.zpa_provisioning_key,
     local_file.user_data_file,
```

Comment:
@willguibr this file is actually not needed and we ran into a weird terraform/aws dependency problem with customers that weren't storing this file after the initial run in their CICD. you can just remove both of those lines of local_file in depends_on for all examples reference